web-components/src/mixins/FocusTrapMixin.ts

```diff
@@ -364,11 +366,14 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
       this.addEventListener("keydown", this.handleKeydownFocusTrap);
       this.addEventListener("focus-visible", this.handleFocusVisible as EventListener);
       const self = this;
-      document.addEventListener("on-component-update", () => {
+      document.addEventListener("on-widget-update", () => {
```

Comment:
please don't change event name, and where "on-widget-update" & "deactivate-focus-trap" is dispatched?

Reply:
This name change was done by Parag, on-widget-update is dispatched from desktop(TransferModalContext.tsx -> on menu overlay open). @parsoman